octodns/provider/route53.py

```diff
@@ -221,9 +221,12 @@ class Route53Provider(BaseProvider):
         access_key_id:
         # The AWS secret access key
         secret_access_key:
+        # The AWS session token
```

Comment:
Probably a good idea to indicate that this one is optional or otherwise call out that `session_token` is only required if you need it. Mainly don't want to confuse people and make them think they have to come up with one to use the provider.